app/modules/MenuCustom/components/ItemCustomizationDialog.tsx

```
@@ -0,0 +1,100 @@
+import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogClose } from "~/components/ui/dialog";
+import { Button } from "~/components/ui/button";
+import type { MenuItem } from "~/store/menuItemState/menuItem.types";
+import { useState } from "react";
+
+const ItemCustomizationDialog = ({
+  showItemDialog,
+  setShowItemDialog,
+  selectedItem,
+  setSelectedItem,
```

ESLint

[error] 'setSelectedItem' is defined but never used.

(@typescript-eslint/no-unused-vars)